src/main/java/com/lambdaschool/oktafoundation/config/OktaAuthSecurityConfig.java

```diff
@@ -49,6 +49,8 @@ protected void configure(HttpSecurity http) throws Exception
                 .antMatchers("/clubusers/**").hasAnyRole("ADMIN", "CD")
                 .antMatchers(HttpMethod.POST,"/memberreactions/**").authenticated() //YDP can post reactions
                 .antMatchers("/memberreactions/**").hasAnyRole("ADMIN","CD") //YDP cannot GET reactions
+                .antMatchers(HttpMethod.GET,"/memberreactions/alert").authenticated()
+                .antMatchers("/memberreactions/alert").hasAnyRole("ADMIN","CD","YDP")
```

Comment:
Looks like this might need to be changed to match the endpoint that was created /memberreactions/memberreactions/alerts

Reply:
Changed the other way around within the controller so that it would match these endpoints